src/sanitizers.rs

```diff
@@ -238,4 +281,32 @@ mod tests {
         let result = strip_stuff_between_messages(input);
         assert_eq!(result, expected);
     }
+
+    #[test]
+    fn excess_tag86_are_stripped() {
+        let input = "\
+            :20:vvvvv\r\n\
+            :86:hello\r\n\
+            multi line string\r\n\
+            here is ok\r\n\
+            but not when\r\n\
+            it is way too many\r\n\
+            lines\r\n\
+            in fact i should be here\r\n\
+            and i shouldnt either\r\n\
+            :62F:C123EUR321,98\r\n\
+        ";
```

Comment:
Please extend this test case to show that this works twice. So if you have a statement that has two overlong tag 86 it will correctly truncate both and it won't misbehave in that case.